internal/provider/provider.go

```diff
@@ -3,14 +3,14 @@ package provider
 import (
 	"context"
 	"fmt"
-	"github.com/hashicorp/terraform-provider-scaffolding-framework/internal/dvc_oauth"
-	"os"
-
 	dvc_mgmt "github.com/devcyclehq/go-mgmt-sdk"
 	dvc_server "github.com/devcyclehq/go-server-sdk"
 	"github.com/hashicorp/terraform-plugin-framework/diag"
 	"github.com/hashicorp/terraform-plugin-framework/tfsdk"
 	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-provider-scaffolding-framework/internal/dvc_oauth"
```

Comment:
Whoop - I think this was IDE autoformatting to alphabetical. Can undo if desired.

Reply:
Also fine - depends on which IDE you use that does or doesn't use this gofmt opt